11_install_agent/20_install_agent_rhel.txt

```diff
@@ -0,0 +1,39 @@
+=== Install Rudder Agent on RedHat-like systems
+
+Download the package applicable to your version of RedHat/CentOS and to its
+architecture on 
+
+----
+
+http://www.rudder-project.org/rpm-3.0/RHEL_<version>/
+
+----
+
+Or you can define a yum repository, replacing <version> with your actual EL version:
```

Comment:
Same as above, please don't use variables for the user to change in the commands